docs/4.operation/5.rancher.md

```diff
@@ -0,0 +1,22 @@
+## Rancher
+
+FIWARE Ops team is currently deploying a customised version of
```

Comment:
Check this sentence, is currently deploying or has deployed... (and where?)

Reply:
@flopezag I recall Rancher was deployed at some point, but I haven't heard anything about it in a while, so I'm not certain of the status on this.

Reply:
In a sentence bellow, I think that it was written that it is still not deployed. Nevertheless, it is a question to certify the status of the deployment of Rancher inside the FIWARE Lab.

Reply:
Confirmed that Rancher is up, updated with link

Reply:
LGTM